### ISSUE TITLE
Hangs when building on "Network mode" on Mac M1

### ISSUE THREAD

**Issue by samstickland** [buildpacks/pack#1577]
### Summary

Hangs when building on "Network mode". Doesn't response to Ctrl-C. Output below is with verbose mode:

```
Host Settings:
  Binds: pack-cache-library_hubbado-web_latest-34cd45190518.build:/cache /var/run/docker.sock:/var/run/docker.sock pack-cache-library_hubbado-web_latest-34cd45190518.launch:/launch-cache pack-layers-rbypfkctuj:/layers pack-app-amzbnwnhdt:/workspace
  Network Mode:
```

---

##### Steps

1.  pack config default-builder heroku/builder:22
2. pack -v build web

##### Current behavior

Hangs on Network Mode:

##### Expected behavior

Image to be built successfully

---

##### pack info

Pack:
  Version:  0.28.0+git-b10e41f.build-3729
  OS/Arch:  darwin/arm64

Default Lifecycle Version:  0.15.2

Supported Platform APIs:  0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10

Config:
  default-builder-image = "[REDACTED]"

##### docker info

Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc., v0.7.1)
  compose: Docker Compose (Docker Inc., v2.2.3)
  scan: Docker Scan (Docker Inc., v0.17.0)

Server:
 Containers: 1
  Running: 0
  Paused: 0
  Stopped: 1
 Images: 22
 Server Version: 20.10.12
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 7b11cfaabd73bb80907dd23182b9347b4245eb5d
 runc version: v1.0.2-0-g52b36a2
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.10.76-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: aarch64
 CPUs: 5
 Total Memory: 1.942GiB
 Name: docker-desktop
 ID: ARQV:JBDN:KACP:NUHW:XNWP:A6Y3:TTCT:EJCM:VMW7:RTWA:636O:3TBU
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http.docker.internal:3128
 HTTPS Proxy: http.docker.internal:3128
 No Proxy: hubproxy.docker.internal
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  hubproxy.docker.internal:5000
  127.0.0.0/8
 Live Restore Enabled: false

**Comment by robertwol** [comment]
Confirm! the same on Mac Intel.
It hang for some time then it kicks in again. the whole build may take up to 8 minutes.

**Comment by jjbustamante** [comment]
Hi @samstickland , @robertwol !

Could you give it a try to the pack 0.30.0-RC2 and double checked if this issues was solved in that version? the issue #1732 is included there

**Comment by jjbustamante** [comment]
Closing this issue based on confirmation from @robertwol  

I forgot to followed up.
